rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommandTest.java

```diff
@@ -126,15 +127,15 @@ public void testFilePathProvided() throws Exception {
         throw stack.get(0);
     }
 
-    @Test(expected = CLIException.class)
+    @Test(expected = JobCreationRestException.class)
     public void testInvalidFilePathProvided() throws Exception {
 
         params = new String[1];
-        params[0] = "/file/path/non/valid/a.xm";
+        params[0] = "/src/test/java/config/c.xm";
```

Comment:
Just need to know why there is no reference to System.getProperty("user.dir") as you did in testNonExistingFilePathProvided ..